osbenchmark/resources/custom-operations.json.j2

```diff
@@ -0,0 +1,16 @@
+    {
+      "name": "index-append",
+      "operation-type": "bulk",
+      "bulk-size": {{bulk_size}},
+      "ingest-percentage": {{ingest_percentage}}
+    },
+    {
+      "name": "search",
+      "operation-type": "search",
+      "index": {{ indices | map(attribute='name') | list | join(',') | tojson }},
+      "body": {
+        "query": {
+          "match_all": {}
+        }
+      }
+    }
```

Comment:
We need to include a way to add custom queries. Some users might use the `--custom-queries` parameter in OSB and attach a JSON file containing queries they plan to run. We need to add those here dynamically